man/man5/puppet.conf.5

```diff
@@ -1105,7 +1105,7 @@ crit
 .IP "" 0
 .
 .SS "logdest"
-Where to send log messages\. Choose between \'syslog\' (the POSIX syslog service), \'eventlog\' (the Windows Event Log), \'console\', or the path to a log file\.
+Where to send log messages\. Choose between \'syslog\' (the POSIX syslog service), \'eventlog\' (the Windows Event Log), \'console\', or the path to a log file\. Multiple destinations can be set using a comma separated list (eg: \fB/path/file1,console,/path/file2\fR)
```

Comment:
From what i remember man files conflicts should be ignored at merge-up as they will be autogenerated for each stream. Might be wrong about this

Reply:
I was confused about this also as there weren't conflicts regarding them so I left them be 😅

Reply:
The automatic merge driver ignores the man files, so they should be ignored on manual merge-ups as well.